fury/actor_text.py

```diff
@@ -0,0 +1,101 @@
+"""Module that provide actors to render."""
+
+import warnings
+import os.path as op
+import numpy as np
+import vtk
+
+from fury.shaders import load, shader_to_actor, attribute_to_actor
+from fury.utils import (
+    rgb_to_vtk, get_actor_from_primitive,  one_chanel_to_vtk)
+import fury.primitive as fp
+from fury import text_tools
+
+
+def bitmap_labels(
```

Comment:
I don't know if is a good name. 
Maybe 
`billboard_labels` is more appropriate